aiidalab/registry/web.py

```diff
@@ -35,28 +34,23 @@ def copy_static_tree_from_path(base_path, static_path):
 
 
 def _walk_pkg_resources(package, root):
-    paths = pkg_resources.resource_listdir(package, root)
-    for path in paths:
```

Comment:
Huh, it looks like you actually fixed a bug here where we would iterate over some directories twice before?
I'll verify a bit more, but the new implementation looks correct. I'll try to add a regression test for this.

Reply:
When writing a unit test for this, I would recommend printing `paths` here and comparing the old and new versions, they are not the same! (although the new one seems more correct as it doesn't duplicate the paths).